src/panel_settings.ts

```diff
@@ -48,6 +49,14 @@ export class Indicator {
 
         this.toggle_tiled = tiled(ext)
 
+        this.toggle_stack_tabs_buttons = toggle(
+            _("Show Tab Buttons for Stacked Windows"),
```

Comment:
Let's call them `Tabs` instead of `Tab Buttons`, the word "buttons" sounds like it's something on the tabs.